passvault/passvault.go

```diff
@@ -35,6 +36,26 @@ const (
 	ECCRecord = "ECC"
 )
 
+const schema = `
```

Comment:
It might be simpler to have a schema where you just have a single string and you marshal your data structure into that, using a JSON+base64 approach like RO does elsewhere. That way, you can basically reuse the rest of the code without modifying it and you won't need to pack/unpack all these columns.